### PR TITLE
Movies search

### PR DESCRIPTION
**Pull request by Kestutis1993**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -7,79 +7,103 @@ class App extends Component {
...
-          display: 'block',
-          paddingLeft: 16,
-          paddingBottom: 8,
-          paddingTop: 8,
-          width: '50%',
-        }} value={this.state.text} onChange={this.searchChangeHandler.bind(this)} placeholder="Enter movie name"/> 
-            <ul className="list">
-              {movies.map(movie => 
+
+  handelSearchResults() {
```
Comment (giesir) [inline src/App.js:52]: This code is rendering something, `renderSearchResults` may be a more appropriate name for it.

File: src/App.js [hunk 2]
```diff
@@ -7,79 +7,103 @@ class App extends Component {
...
+    const {movies, searchResults} = this.state;
+    if (searchResults) {  
+      return <div className="results_block">
+        <div className="results_box">
+          <i className="fas fa-film fa-lg" />
+          <div className="input_box">
+            <input value={this.state.text} 
+              onChange={this.searchChangeHandler.bind(this)}
+            />
+            <p className="plcholder">Enter a movie name</p>
```
Comment (giesir) [inline src/App.js:62]: `plcholder` -> `placeholder`

**Review by giesir — COMMENTED** [review]
A few global notes:

1. Artefacts, such as images, should not reside within the code repository - they should be hosted somewhere and distributed over a CDN, if applicable.
2. Why is App.js performing business related functionality? The purpose of the App component is to render the root of an application. All functionality should reside within their own component hierarchy.

**Review by Kestutis1993 — COMMENTED** [review]
I fixed all mistakes those were before this commit.

**Pull request merged**